website/docs/contributing/writing-guide.md

```diff
@@ -60,6 +60,10 @@ To link to a file in a parent directory, prefix with `../` to go up one director
 
 To link to file in a subdirectory, specify the file path along with its respective directory or directories, such as: `subdirectory/file.md`.
 
+To link to a directory or category, prefix the directory name with `/category/`.
+
+- `[configuration directory](/category/configuration/)`
```

Comment:
Does it have to be an absolute link? Sounds strange to demonstrate the absolute link in the section that is all about relative paths.

Reply:
It works equally like this `(../../category/configuration/)`. Seems the category path is structured higher up in the docs hierarchy.

Reply:
Could it be a link to the directory instead?

Reply:
Do you mean like this (https://docs.ferretdb.io/category/configuration/)?

Reply:
No, I don't think we should use the domain name anywhere

Reply:
I don't understand the last question. Should I make it a clickable link instead of just having it in the code block?

Reply:
> To link to a directory or category

What is a link to a directory? Does it work? If yes, that should be a separate point from the category

Reply:
There's no direct way to link to the directory except the ones above. Any link to a directory would involve getting the autogenerated cards in `_category_.yml`.